### PR TITLE
created Bucket.md

### PR DESCRIPTION
**Pull request by Ravineel**
In the unordered map, bucket.md file is made and also in the snippet bucket.CPP file.

### CODE REVIEW

File: snippets/unordered_map/bucket.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include <bits/stdc++.h>
```
Comment (Delta456) [inline snippets/unordered_map/bucket.cpp:1]: Please refrain from using this. It is recommended to use separate header files for specific purpose.

File: snippets/unordered_map/bucket.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include <bits/stdc++.h> 
+using namespace std;
```
Comment (Delta456) [inline snippets/unordered_map/bucket.cpp:2]: Do not use this as well

**Comment by Delta456** [comment]
Closing but can be merged later if the guidelines are followed.

**Comment by Ravineel** [comment]
sure will do

**Pull request closed without merging**